### DISCUSSION TITLE
NSFX (Name suffix)

### DISCUSSION THREAD

**Discussion by hartenthaler** [fisharebest/gedcom-name#16] [General]
The GEDCOM standard 7.0 says

> Text which appears on a name line after or behind the given and surname parts of a name.

I'm using that for example to add "jr." or "sr." to a name. If the same name is used in more than two generations I'm using roman numbers (starting in the oldest generation with "I", then "II" and so on).

Using this is maybe not correct for birth names, because "Rudolf /Hartenthaler/ sr." was not born with the suffix "sr.". He got this suffix when his son "Rudolf Hartenthaler jr." was born in 1950.

What will the new GEDCOM NAME standard recommend in such cases? Using two NAME records?

> 1 NAME Rudolf /Hartenthaler/
> 2 TYPE BIRTH
> 1 NAME Rudolf /Hartenthaler/ sr.
> 2 TYPE AKA
> 2 DATE FROM 1950

A similar topic is the suffix "the Elder", for example: "Lucas Cranach the Elder".

**Comment by webtrees-pesz** [comment]
Hermann,

the same here. Each NAME tag should represent the name as found in the cited source.
For output in lists and diagrams, a different name tag should be used. In such a name tag you can define the names as you want to display them.

**Reply by Tim-Partridge** [reply]
I think using the source format is important. Though family historians often like to create a universal unique name for each person. This scheme can become difficult as some personal names become popular in a family and may be shared by cousins. Further if the researcher dubs someone John Smith I, then discovers their father was also a John Smith a renumbering is called for!
Is there a way to distinguish researcher created nick names from those in a record? 
In societies where women adopt their husband's name on marriage it can be difficult to know the original surname of a woman. Some researchers put in their married name, others leave blank or put unknown. This complicates comparing two trees to see if they match enough to represent the same families.

**Comment by hartenthaler** [comment]
My example using roman generation numbers is even more complex. In my tree, there is "Josiah Shipp I" (born in 1664). His birth name is "Josiah Shipp". He got the suffix I when his grandson "Josiah Shipp III" was born in 1717. At that time he was already 13 years dead. Do we like to have something like 

> 1 NAME Josiah /Shipp/ III
> 2 TYPE AKA
> 2 DATE FROM 1717
> 1 DEAT
> 2 DATE 1704

Would it be allowed or makes sense that events or name dates are after the death of a person?

**Comment by hartenthaler** [comment]
@webtrees-pesz 
> Each NAME tag should represent the name as found in the cited source.

Ohh!!! There was never ever a source that shows "Rudolf Hartenthaler sr." or even worse "Josiah Shipp VI". Those name suffixes were invented internally in the family or many generations later by some genealogical researchers. 

The same is the case for many nicknames in my tree. Primary sources for nicknames are seldom. Following your statement strictly, it would be not allowed to add nicknames or name suffixes to birth names. Nicknames are used sometimes from day 1 in the life of a person, but most nicknames a created later.

Or I have to invent additionally a source "Josiah Shipp VI was invented by the genealogical researcher Hermann Hartenthaler in 2022" or "Rudolf Hartenthaler sr. was used by his daughter Elisabeth at a family event in 1970".

**Reply by Tim-Partridge** [reply]
I have seen a marriage record (in Devon, England) where someone is referred to in the format John Smith Jr. This distinguishes them from the other John Smith in the area. (Who was his father.) I have seen on a web site mentions that two John Smiths might not be closely related, just living in the same area and still be called Sr and Jr. If they had been living in different areas, the record maker might not have bothered or even have been aware of the other John Smith.

**Reply by albertemmerich** [reply]
I have had church records in Germany like Johann Heinrich Emrich 12. , as 11 other Johan Heinrich Emrich were living at the same time in the same village. Problem is, that these persons were renumbered in case one of them died, so at different events the suffix number used in the original source we have different suffixes, and by that different NAME !

**Comment by hartenthaler** [comment]
Following the argument of @webtrees-pesz it should be

> 1 NAME Josiah /Shipp/
> 2 TYPE BIRTH
> 2 SOUR @S1@
> 1 DEAT
> 2 DATE 1704
> 1 NAME Josiah /Shipp/ III
> 2 TYPE AKA
> 2 DATE FROM 2022
> 2 SOUR @S2@
> 0 @S2@ SOUR
> 1 TITL Genealogical research report
> 1 AUTH Hermann Hartenthaler
> 1 PUBL https://ahnen.hartenthaler.eu/tree/ahnen/individual/I6365/Josiah-Shipp-III-I6365
> 1 TEXT The name suffix "III" of Josiah Shipp was invented by the genealogical researcher Hermann Hartenthaler in 2022. 

Is this correct? Is that the way everybody should do it? Is there any genealogical program that guides its users to do it that way?

**Comment by Norwegian-Sardines** [comment]
> Is this correct? Is that the way everybody should do it? Is there any genealogical program that guides its users to do it that way?

First, I would say that since many genealogical programs don’t support all (or any) of the current NAME subtags, if they do make suggestions about entering data it is to support their database design not the data’s eventual exportation via GEDCOM.

Second, many family historians are interested in a limited amount of data entry and/or are guided by a personal or family pressure to use only names the reader is familiar with, therefore using suffixes like Jr, Sr, III,  or prefixes like Dr., Colonel, Reverend, Mrs., Duke, etc. or including Nick Names, Married Names as part of the primary name are needed to support these desires.

I would hope that any GEDCOM I receive has some thought (and control) behind it regarding the use of NAMEs but I can’t expect it.   GEDCOM on the other hand should provide a framework that allows data to be logically built so that it can be used in other programs and by other researchers who have a more controlled sense of data. 

While I would not use the actually wording of your text, the concept of recording names in a historical research project “as entered” has merit and should be considered.  Remember, since we are talking about historical documents, and that the possibility of errors in their recording exist, any name found in our research is provisional.  Therefore we should not be too hasty to make a conclusion as to what name is the correct one or should ultimately be used as the primary name.
